pydub/audio_segment.py

```diff
@@ -562,6 +562,18 @@ def set_channels(self, channels):
                                'channels': channels,
                                'frame_width': frame_width})
 
+    def split_to_mono(self):
+        if self.channels == 1:
+            return self
+
+        left_channel = audioop.tomono(self._data, self.sample_width, 1, 0)
+        right_channel = audioop.tomono(self._data, self.sample_width, 0, 1)
+
+        return [self._spawn(data=left_channel,
```

Comment:
overrides should be: `{'channels': 1, 'frame_width`: self.sample_width}` and in the tests the `frame_count` should not change.

A "frame" contains a sample for each channel (all of which are "played" simultaneously)